instant-xml/tests/all.rs

```diff
@@ -4,10 +4,12 @@ use instant_xml::{FromXml, ToXml};
 struct Unit;
 
 #[derive(Debug, Eq, PartialEq, ToXml)]
-#[xml(namespace("URI"))]
+#[xml(namespace("URI", bar = "BAZ", foo = "BAR"))]
 struct StructWithNamedFields {
     flag: bool,
+    #[xml(namespace("bar"))]
     string: String,
+    #[xml(namespace("typo"))]
```

Comment:
Same here, though I think we should probably yield an error on serialization (not while deriving, since it could be that a parent element defines it for us) if we encounter a namespace prefix that hasn't been defined.

Reply:
Agree, but I would do the error handling as a separate PR.